test/common/config/http_subscription_impl_test.cc

```diff
@@ -42,6 +42,24 @@ TEST_F(HttpSubscriptionImplTest, BadJsonRecovery) {
   verifyStats(3, 1, 0, 1, 7148434200721666028);
 }
 
+TEST_F(HttpSubscriptionImplTest, ConfigNotModified) {
+  startSubscription({"cluster0", "cluster1"});
+  EXPECT_CALL(random_gen_, random()).WillOnce(Return(0));
+  EXPECT_CALL(*timer_, enableTimer(_));
+
+  verifyStats(1, 0, 0, 1, 0);
```

Comment:
I think you are still expecting there to be a failure here, can you look into  the CI failures? Will look again when green.
/wait

Reply:
I have no idea which lines are expecting failures... I'm assuming it's the `EXPECT_CALL` lines. Problem is, I can't test any of this locally, as I'm running on OSX. All I can do is push commits and see if it goes green, guess I'll do so.

Reply:
@cetanu by saying: 

```cpp
verifyStats(1, 0, 0, 1, 0);
```
you're expecting 1 failure.

```cpp
  virtual void verifyStats(uint32_t attempt, uint32_t success, uint32_t rejected, uint32_t failure,
                           uint64_t version) {
```

but here, we don't? Probably update it like:

```diff
-  verifyStats(1, 0, 0, 1, 0);
+  verifyStats(1, 0, 0, 0, 0);
```

And also for all the `verifyStats`s in that block.

Reply:
@cetanu why can't you test this on OS X? This test shouldn't have any platform dependencies. Have you look at the OS X flow at https://github.com/envoyproxy/envoy/tree/master/bazel?